src/tcprewrite.c

```diff
@@ -229,7 +229,7 @@ rewrite_packets(tcpedit_t *tcpedit, pcap_t *pin, pcap_dumper_t *pout)
     COUNTER packetnum = 0;
     int rcode;
 #ifdef ENABLE_FRAGROUTE
-    int frag_len, i, proto;
+    int frag_len, proto;
```

Comment:
This will break compile if using `./configure --enable-debug` and when using FRAG_ROUTE. I have been working on a fix in 4.2.